src/atoms/Radio/__tests__/Radio.web.test.js

```diff
@@ -0,0 +1,358 @@
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('size', () => {
+    it('should render a large radio input', () => {
+      const { container } = renderWithTheme(
+        <Radio value="1" size="large">
+          <Radio.Option value="1" />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should render a medium(default) radio input', () => {
+      const { container } = renderWithTheme(
+        <Radio value="1" size="medium">
+          <Radio.Option value="1" />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should render a small radio input', () => {
+      const { container } = renderWithTheme(
+        <Radio value="1" size="small">
+          <Radio.Option value="1" />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('variantColor', () => {
+    it('should render a radio with default variantColor if variantColor not provided', () => {
+      const { container } = renderWithTheme(
+        <Radio value="1">
+          <Radio.Option value="1" />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should render a radio with provided variantColor', () => {
+      const { container } = renderWithTheme(
+        <Radio value="1" variantColor="shade">
+          <Radio.Option value="1" />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('focus', () => {
+    describe('with defaultValue', () => {
+      it('should render focus styles on radio when radio is not checked', () => {
+        const { container, getByRole } = renderWithTheme(
+          <Radio defaultValue="1">
+            <Radio.Option value="2" />
+          </Radio>,
+        );
+        const radio = getByRole('radio');
+        radio.focus();
+        expect(radio).toHaveFocus();
+        expect(container).toMatchSnapshot();
+      });
+
+      it('should render focus styles on radio when radio is checked', () => {
+        const { container, getByRole } = renderWithTheme(
+          <Radio defaultValue="1">
+            <Radio.Option value="1" />
+          </Radio>,
+        );
+        const radio = getByRole('radio');
+        radio.focus();
+        expect(radio).toHaveFocus();
+        expect(container).toMatchSnapshot();
+      });
+    });
+
+    describe('with value', () => {
+      it('should render focus styles on radio when radio is not checked', () => {
+        const { container, getByRole } = renderWithTheme(
+          <Radio value="1">
+            <Radio.Option value="2" />
+          </Radio>,
+        );
+        const radio = getByRole('radio');
+        radio.focus();
+        expect(radio).toHaveFocus();
+        expect(container).toMatchSnapshot();
+      });
+
+      it('should render focus styles on radio when radio is checked', () => {
```

Comment:
Duplicate of above test.

Reply:
don't think its duplicate. One is when radio is checked and one is when its not checked.

Reply:
Fix this as discussed.